tsconfig.json

```diff
@@ -5,7 +5,6 @@
         "downlevelIteration": true,
         "esModuleInterop": true,
         "isolatedModules": true,
-        "importsNotUsedAsValues": "error",
```

Comment:
@prezly/slate-types:build: error TS5101: Option 'importsNotUsedAsValues' is deprecated and will stop functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.
@prezly/slate-types:build:   Use 'verbatimModuleSyntax' instead.
@prezly/slate-types:build:  ELIFECYCLE  Command failed with exit code 2.
@prezly/slate-types:build:  ELIFECYCLE  Command failed with exit code 1.
@prezly/slate-types:build: ERROR: command finished with error: command (/home/runner/work/slate/slate/packages/slate-types) pnpm run build exited (1)
@prezly/slate-lists:build:  ELIFECYCLE  Command failed.
@prezly/slate-lists:build:  ELIFECYCLE  Command failed.